client/components/GameDescription.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import Logo from './logo.jsx';
+import Tags from './tags.jsx';
+
+const GameDescription = (props) => (
+  <div id="reviews_container">
+    <Logo logoURL={props.logoURL} />
+    <dl>
+      <dt className="game_description">
+        {props.description}
+      </dt><br />
```

Comment:
Are these `<br />` tags necessary? They seem inconsistent and appear between the `<dt>`s sometimes and sometimes not. If they're there simply for presentation I think using margins in the styling would be preferable.

Reply:
They are only used for styling. Will try to refactor this.